src/vicsetup/libvicsetup/argon2/Argon2.sln

```diff
@@ -0,0 +1,158 @@
+﻿
```

Comment:
We don't build anything with Visual Studio, we probably don't want visual studio solution files committed.

Reply:
Right. This is part of the argon2 distro. I will create a submodule for it.

Reply:
To resolve this, I added an argon2 submodule.